modulefiles/hera.gnu/fv3

```diff
@@ -30,6 +30,7 @@ module use -a /scratch1/BMC/gmtb/software/ufs-stack-20200909/gnu-9.2.0/mpich-3.3
 
 module load libpng/1.6.35
 module load netcdf/4.7.4
+#module load pio/2.5.1
```

Comment:
This should be ready. If my update to use ufs-stack for hera.gnu doesn't go in before your commit, I can send you the hash to cherry-pick. Results don't change when switching from the current libraries to the ufs-stack for hera.gnu

Reply:
@climbfuji 
@Hang-Lei-NOAA installed the hpc-stack with GNU 9.2.0 and MPICH 3.3.2 along side the Intel stack on Hera
The GNU regression tests passed on Hera with GNU.
I have updated the modulefile for `hera.gnu` to use this update

Reply:
Excellent, thanks! This ball is finally off my court ;-)